src/certifi.erl

```diff
@@ -26,9 +26,13 @@ cacertfile() ->
 -spec cacerts() -> [binary(),...].
 cacerts() ->
     ct_expand:term(
-      lists:reverse(
-        [Der || {ok,Bin} <- [file:read_file(cacertfile())],
-                {'Certificate',Der,_} <- public_key:pem_decode(Bin)
-        ]
+      lists:foldl(
+        fun ({'Certificate', Der, _}, Acc) ->
+                [Der | Acc]
+        end,
+        [],
+        (fun ({ok, Bin}) ->
+                 public_key:pem_decode(Bin)
+         end)(file:read_file(cacertfile()))
```

Comment:
^ The new expression is very convoluted; this is due to limitations in the kind of expressions `ct_expand:term/1` accepts. But failed patterns will no longer be ignored 🙂